sasmodels/resolution.py

```diff
@@ -581,32 +580,32 @@ def romberg_slit_1d(q, width, length, form, pars):
         width = [width]*len(q)
     if np.isscalar(length):
         length = [length]*len(q)
-    _int_w = lambda w, qi: eval_form(sqrt(qi**2 + w**2), form, pars)
-    _int_l = lambda l, qi: eval_form(abs(qi+l), form, pars)
+    _int_w = lambda wi, qi: eval_form(sqrt(qi**2 + wi**2), form, pars)
+    _int_l = lambda li, qi: eval_form(abs(qi+li), form, pars)
```

Comment:
I updated the romberg_slit_1d function to use length, width rather than width, length and reversed the use of length and width within the function. However, this led me to a similar change for slit_extend_q that was still using width, length. It appeared as though slit_extend_q was called in different ways so I've tried to correct this in the appropriate places. The tests that are still implemented are passing, but this will need careful review.